common/liotarget

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+#
+# Copyright (c) 2017 Virtuozzo Inc
+# All Rights Reserved.
+#
+# Written by Dmitry Monakhov <dmonakhov@openvz.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it would be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write the Free Software Foundation,
+
+#
+# Functions for Linux-IO Target manipulation
+#
+
+_require_liotarget()
```

Comment:
This should be `_have_liotarget`, and it should work like the other `_have_*` examples in common/rc. Namely, `_notrun` won't work for blktests, you need to return 0 if the requirements are met or set `SKIP_REASON` and return 1 if not.